src/applications/personalization/profile360/vet360/actions/transactions.js

```diff
@@ -29,6 +29,42 @@ function recordProfileTransaction(event, fieldName) {
   }
 }
 
+export function initializeUserToVet360() {
+  return async (dispatch) => {
+    const method = 'POST';
+    const fieldName = 'initialization';
+
+    const options = {
+      method,
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    };
+
+    try {
+      dispatch({
+        type: VET360_TRANSACTION_REQUESTED,
+        fieldName,
+        method
+      });
+
+      const transaction = isVet360Configured() ? await apiRequest('/profile/initialize_vet360_id/', options) : await localVet360.createTransaction();
+
+      dispatch({
+        type: VET360_TRANSACTION_REQUEST_SUCCEEDED,
+        fieldName,
+        transaction
+      });
+    } catch (error) {
+      dispatch({
+        type: VET360_TRANSACTION_REQUEST_FAILED,
+        error,
+        fieldName
+      });
+    }
+  };
+}
+
```

Comment:
I was debating how we should go about this. I think when we refactor we ultimately need a generic transaction function that we can leverage here, and during the update/delete action creators.